docs/content/guides/primer_local_time.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Moving Away From `Primer::LocalTime`
+---
+
+This guide will show you how to upgrade from the now deprecated
+[`Primer::LocalTime`](https://primer.style/view-components/components/localtime)
+to the latest [`Primer::Beta::RelativeTime`](https://primer.style/view-components/components/beta/relativetime)
+component.
+
+## A Migration Example
+
+The most common use case of the `LocalTime` component can be migrated with only
+a few minor changes.
+
+For example, if the `LocalTime` was set up in this way:
+
+```rb
+<%= Primer::LocalTime(datetime: Time.now, initial_text: Time.now.iso8601) %>
+```
+
+It can be migrated and removing `initial_text` and setting an empty `prefix`.
+
+```rb
+<%= Primer::Beta::RelativeTime(datetime: Time.now, prefix: "", threshold: "PT0S") %>
+```
+
+The `RelativeTime` component defaults to the `iso8601` format and does not need
+to be specified directly.
+
+The `threshold` value is an [ISO-8601 "duration"](https://en.wikipedia.org/wiki/ISO_8601#Durations) that tells the `RelativeTime`
+component to display the absolute date/time, instead of relative time
+description. The example of `PT0S` says to switch to absolute time display
+starting zero (0) seconds ago. In practice, this means it will always display
+the absolute time. With the `LocalTime` component, `PT0S` was the default
+threshold. The `RelativeTime` component does not have a default, however, and
+it should should be specified if needed.
```

Comment:
```suggestion
threshold. The `RelativeTime` component defaults to `P30D`, however, and
it will need to be zeroed out to always display a datetime.
```